crates/starfish/core/src/error.rs

```diff
@@ -125,8 +125,9 @@ pub(crate) enum ConsensusError {
     #[error("Insufficient stake from parents: {parent_stakes} < {quorum}")]
     InsufficientParentStakes { parent_stakes: Stake, quorum: Stake },
 
+    #[allow(dead_code)]
```

Comment:
As Ilia suggested in other places

```suggestion
    #[expect(dead_code)]
```